packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -480,6 +483,14 @@ function SelectedTransactionsButton({
     };
   }, [selectedItems]);
 
+  let ambiguousDuplication = useMemo(() => {
+    let transactions = [...selectedItems].map(id => getTransaction(id));
```

Comment:
If anyone has any suggestions for how this ambitiousDuplication fn can be improved, would love to hear them, seems a little unfortunate that we have to get ALL the selected transactions to determine if we can enable this functionality or not.

Reply:
You could show an error message after hitting the duplicate button (since the case where the duplication is invalid is unlikely, right?)

It might also make sense to duplicate just the top level transaction if it is selected along with some of its children (since that most closely marches the expected behavior IMO).

Reply:
I think an error message makes more sense here, agreed.

I can see either way with this behavior to be honest. I think overall duplicating child transactions seems like a bit of an unnecessary feature in my opinion, especially with updating the parents amount automatically, it just seems a bit backwards to how I personally go about splitting a transaction. 

Would be super interested to hear if anyone does have a workflow where duplicating individual child transactions would be useful!

Reply:
> Would be super interested to hear if anyone does have a workflow where duplicating individual child transactions would be useful!

Maybe start by not duplicating child transactions and wait for someone to complain?

Reply:
Yeah, I think thaat sounds like a better plan! I'll go ahead remove the child duplication code, easy to add back in later if it is a desired feature

@MatissJanis, wanted to ping you here since we spoke about this earlier on Discord!

Reply:
Sounds good